ytfzf

```diff
@@ -871,12 +877,15 @@ get_search_query () {
 		if [ "$is_ext_menu" -eq 1 ]; then
 			#when using an external menu, the query will be done there
 			search_query=$( : | eval "$external_menu" )
+        elif [ $enable_search_hist_menu -eq 1 ]; then
+            search_history_menu
 		else
 			#otherwise use the search prompt
 			printf "$search_prompt"
 			read -r search_query
 		fi
 		[ -z "$search_query" ] && clean_up && exit 0
```

Comment:
this code block is the else block of the `if [ "$is_ext_menu" -eq 1 ]` check, because when there is no search query it checks if the user is using an external menu. In that case it will prompt the search query there, otherwise it will use the terminal with `read`

Reply:
also you might want to move `set_search_history` outside of the if check, because if there is no search and the user just runs, `ytfzf` then gets the search prompt, it won't store the search.

Reply:
> also you might want to move `set_search_history` outside of the if check, because if there is no search and the user just runs, `ytfzf` then gets the search prompt, it won't store the search.

`set_search_history` is outside of the if check?

Reply:
cool, if you're ready for it to be merged add a reaction to this, cause it looks good to me

Reply:
I think I'll add a few more basic user options before merge.

Reply:
Do you want me to add anything to the docs? Or add the options to the example conf.sh?
Otherwise I am ready for the PR to be merged.

Reply:
you should add 
`enable_search_hist`, `enable_search_hist_menu`, and `allow_empty_search_hist` to the `OPT VARIABLES` section in `docs/conf.sh`
and add
`search_history_prompt`, and `search_history_file` to `MSIC/Other` or near where `useragent` is.

Reply:
> set_search_history is outside of the if check?

`set_search_history` needs to be called the line before `function_exists "on_get_search"...` because it doesn't save when you search like `ytfzf search`